src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -142,16 +119,17 @@ export default function Banner() {
         justify="center"
         className={classes.container}
       >
-        <Grid id="back-to-top-anchor" item xs={8}>
+        <Grid id="back-to-top-anchor-mobile" item xs={8}>
```

Comment:
Why `-mobile` on an id?  Seems like something that should only happen in a class you apply, since this will be the same element in every size, no?

Reply:
There are two different anchors. 
The desktop anchor places the post's title on the top of the navbar, the mobile anchor is placing the post's title under the navbar.